commit fde3e5a05714067c4635e4d66637370e29b42037
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-31T10:38:00-04:00 (10 weeks ago)
    
    TRITON-1847 prometheus image should use node v6
    TRITON-1848 update prometheus base image to triton-origin-x86_64-18.4.0
    Reviewed by: Isaac Davis <isaac.davis@joyent.com>
    Approved by: Isaac Davis <isaac.davis@joyent.com>

diff --git a/Makefile b/Makefile
index 6aa002a..7e74359 100644
--- a/Makefile
+++ b/Makefile
@@ -11,11 +11,10 @@
 NAME = prometheus
 
 GO_PREBUILT_VERSION = 1.11.1
-NODE_PREBUILT_VERSION = v6.15.1
+NODE_PREBUILT_VERSION = v6.17.0
 ifeq ($(shell uname -s),SunOS)
-    NODE_PREBUILT_TAG=zone
-    # Allow building on other than triton-origin-multiarch-18.1.0
-    NODE_PREBUILT_IMAGE=1ad363ec-3b83-11e8-8521-2f68a4a34d5d
+    NODE_PREBUILT_TAG=zone64
+    NODE_PREBUILT_IMAGE=c2c31b00-1d60-11e9-9a77-ff9f06554b0f
 endif
 
 ENGBLD_USE_BUILDIMAGE = true
@@ -30,10 +29,10 @@ ifeq ($(shell uname -s),SunOS)
     include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 endif
 
-# triton-origin-multiarch-18.1.0
-BASE_IMAGE_UUID = b6ea7cb4-6b90-48c0-99e7-1d34c2895248
+# triton-origin-x86_64-18.4.0
+BASE_IMAGE_UUID = a9368831-958e-432d-a031-f8ce6768d190
 BUILDIMAGE_NAME = manta-$(NAME)
-BUILDIMAGE_PKGSRC = bind-9.10.7
+BUILDIMAGE_PKGSRC = bind-9.11.8
 BUILDIMAGE_DESC = Triton/Manta Prometheus
 AGENTS = amon config registrar
 
diff --git a/deps/eng b/deps/eng
index d25b8fc..63e056b 160000
--- a/deps/eng
+++ b/deps/eng
@@ -1 +1 @@
-Subproject commit d25b8fc60fb8c649559924870fe3aaf75e7421d5
+Subproject commit 63e056ba42fd426b9314e99d1cd92773f4dcdda9
